core/services/relay/evm/types/types.go

```diff
@@ -192,7 +202,8 @@ type RelayConfig struct {
 	EnableTriggerCapability bool         `json:"enableTriggerCapability"`
 
 	// LLO-specific
-	LLODONID uint32 `json:"lloDonID" toml:"lloDonID"`
+	LLODONID      uint32        `json:"lloDonID" toml:"lloDonID"`
+	LLOConfigMode LLOConfigMode `json:"lloConfigMode" toml:"lloConfigMode"`
```

Comment:
If these are product specific, should they be in the `lloconfig.PluginConfig`?

Reply:
It may be needed for bootstrap jobs as well, hence why I put it here